reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md

```diff
@@ -82,7 +82,28 @@ This cmdlet was introduced in PowerShell 3.0.
 
 ## EXAMPLES
 
-### Example 1: Use a stateful web service
+### Example 1: Send a web request
```

Comment:
This example is invalid in v6 because of the change to .NET Core. Need to review all examples and make sure they still work in v6.